dss-api

```diff
@@ -33,4 +33,4 @@ dss.Config.set_config(dss.DeploymentStage.NORMAL)
 logging.basicConfig(level=args.log_level, stream=sys.stderr)
 
 factory = CLIFactory(project_dir=args.project_dir, debug=args.debug)
-run_local_server(factory, port=args.port, env=os.environ)
+run_local_server(factory, port=args.port, env=os.environ, stage=os.environ["DSS_DEPLOYMENT_STAGE"])
```

Comment:
Is this a new chalice thing?

Reply:
Yep... https://github.com/aws/chalice/commit/d1e3e3f